syntax = "proto3";

message Version
{
    int32 version_id = 1;
    int32 version_number = 2;
    int32 project_id = 3;
    string project_name = 4;
    int32 user_id = 5;
    string contents = 6;
    string operations = 7;
}

message Versions
{
    repeated Version versions = 1;
}